### PR TITLE
Added MSW and SendNFT test

### PR DESCRIPTION
**Pull request by razvantomegea**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-VITE_APP_PERSIST="localStorage"
\ No newline at end of file
+VITE_APP_PERSIST="localStorage"
+VITE_APP_MSW="false"
```
Comment (arhtudormorar) [inline .env:2]: add to vite.d.ts

File: package.json [hunk 2]
```diff
@@ -103,6 +104,7 @@
     "jest-resolve": "29.7.0",
     "mochawesome-merge": "4.3.0",
     "mochawesome-report-generator": "6.2.0",
+    "msw": "1.3.1",
```
Comment (arhtudormorar) [inline package.json:107]: try upgrade?

File: src/pages/Send/Send.tsx [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import { useEffect, useState } from 'react';
+import { getEgldLabel } from '@multiversx/sdk-dapp/utils';
+import { computeTokenDataField } from '@multiversx/sdk-dapp-form/operations/computeDataField';
```
Comment (arhtudormorar) [inline src/pages/Send/Send.tsx:3]: import all from lib

File: src/pages/Send/hooks/useSendForm.ts [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import { getEgldLabel } from '@multiversx/sdk-dapp/utils';
```
Comment (arhtudormorar) [inline src/pages/Send/hooks/useSendForm.ts:1]: lib

File: .env (hunk 1 above)
Comment (razvantomegea) [reply .env:2]: Added in vite as well

**Review by arhtudormorar — APPROVED** [review]

**Pull request merged**